client/lib/mixins/i18n/index.js

```diff
@@ -14,7 +14,7 @@ var debug = require( 'debug' )( 'calypso:i18n' ),
 var config = require( 'config' ),
 	numberFormatPHPJS = require( './number-format' ),
 	emitter = require( 'lib/mixins/emitter' ),
-	interpolateComponents = require( 'lib/interpolate-components' );
+	interpolateComponents = require( 'lib/interpolate-components/src' );
```

Comment:
I too played with this.  I specified a webpack property in the package.json so that you can get rid of the src in the path.  This worked well until I ran the tests which do not utilize webpack.  If I make main 'src' then the package will fail if installed by a use via npm.  Any ideas?  https://webpack.github.io/docs/configuration.html#resolve-packagemains